assets/shaders/custom_material_import.wgsl

```diff
@@ -0,0 +1,2 @@
+// this is made available to the importing module
+const MULT: vec4<f32> = vec4<f32>(1.0, 1.0, 1.0, 0.5);
```

Comment:
This is a weird and unclear variable name. Can we change it to something more semantic?

Reply:
sure, how about `COLOR_MULTIPLIER` or `EXAMPLE_IMPORTED_COLOR_MULTIPLIER` ?

Reply:
Both are fine. The former is probably adequate.